index.html

```diff
@@ -791,6 +791,36 @@ <h3>Devices</h3>
           </dd>
         </dl>
     </section>
+    <section>
+      <h3>Generic</h3>
+        <p>
+The generic domain contains use cases that can apply to any or all of the proceeding domains
```

Comment:
This is not common US English usage of `proceeding`, and I am not sure what is intended by it.  (The sentence also lacks a closing `.`.)

Reply:
I think preceding was intended.